src/consensus/vote.cpp

```diff
@@ -220,48 +215,101 @@ uint64_t VoteManager::getUnverifiedVotesSize() const {
   uint64_t size = 0;
 
   sharedLock_ lock(unverified_votes_access_);
-  std::map<uint64_t, std::unordered_map<vote_hash_t, Vote>>::const_iterator it;
-  for (it = unverified_votes_.begin(); it != unverified_votes_.end(); ++it) {
+  for (auto it = unverified_votes_.begin(); it != unverified_votes_.end(); ++it) {
     size += it->second.size();
   }
 
   return size;
 }
 
+std::vector<std::shared_ptr<Vote>> VoteManager::getVerifiedVotes() {
+  std::vector<std::shared_ptr<Vote>> votes;
```

Comment:
reserve size based on `verified_votes_` size

Reply:
We don't have variable "size" for record unverified/verifed votes in class, since votes will change very often.
And getVerifiedVotesSize() need go through PBFT rounds, steps, and voted value for counting verfied_votes size. And I think that is not good for getting size first and reserve it

Reply:
Fixed, thanks

Reply:
no like this, this is just overhead. after the lock you can do `votes.reserve(verified_votes_.size());`